### DISCUSSION TITLE
What is the largest 🏝️ in the Caribbean?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#996] [Galaxy Trivia] [answered]
**Bonus:** What is the tallest 🏢 there?

**Comment by peterbaccaro** [comment, marked as answer]
**Cuba** is the largest island country in the Caribbean sea, with a total area of almost 111 thousand square kilometres.

The tallest building in Cuba is the **Edificio Focsa** (i.e. Fosca Building), located in Havana. The Edificio Focsa stands at a height of approximately 121 meters (397 feet) and has 39 floors. It was completed in 1956 and has been an iconic landmark in Havana ever since.
